### ISSUE TITLE
Gradient Descent

### ISSUE THREAD

**Issue by jihelhere** [ProjetStageXinHe2022/Combinatorial-Optimization-with-GCNN#9]
1. Scheduler pour learning rate
2. Batch Size ajustable (uniquement si layernorm et LR ne suffisent pas à stabiliser le gradient)
